Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -198,12 +202,29 @@ public struct PackageBuilder {
         guard let pkgConfig = manifest.package.pkgConfig else { return nil }
         return RelativePath(pkgConfig)
     }
+
+    // FIXME: Should this be an extension on FileSystem and placed in Utility?
+    /// Returns path to all the items in a directory.
```

Comment:
I tend to think not. The reason is that I'm not sure how common it is, and it is often inefficient since in my experience clients using this often want to look at all the names and then use the combined path. If they use a function which returns the combined path, they have to do unnecessary extra work to extract out the name again.